src/encode.cpp

```diff
@@ -2613,8 +2617,13 @@ static void encodeReductionLoopBodyAndOutput(
     // t_res[0] = sum(\i. t_input[i / n][i % n] , i < m * n)
 
     // Define this as a splat tensor (num. elems is 1 anyway)
-    t_res = Tensor(t_v.getElemType(), t_v.sum(),
+    // TODO (seongwon): Currently we cover only tensor cases..
+    // TODO: Support memref cases
+    auto outty = newst.regs.get<Tensor>(the_op->getOperands().back());
```

Comment:
Nit: `outty` vs. `outTensor`?

Reply:
Another question: can a reduction loop have multiple outputs?

Reply:
It checked at here (https://github.com/aqjune/mlir-tv/blob/master/src/encode.cpp#L2704-L2706)